schemas/clinvar/tables/VariationArchiveVariation.json

```diff
@@ -0,0 +1,60 @@
+ {
+    "name": "variation_archive_variation",
+    "columns": [
+	{
+	    "name": "variation_archive_variation_id",
+	    "datatype": "int",
+	    "links": [
+		{
+		    "table_name": "variation_archive_variation",
+		    "column_name": "parent_id"
+		},
+		{
+		    "table_name": "clinical_assertion",
+		    "column_name": "variation_archive_variation_id"
+		}
+	    ]
+	},
+	{
+	    "name": "name",
+	    "datatype": "string"
+	},
+	{
+	    "name": "variation_type",
+	    "datatype": "string"
+	},
+	{
+	    "name": "subclass_type",
+	    "datatype": "string"
+	},
+	{
+	    "name": "allele_id",
```

Comment:
Maybe this is a piece that hasn't been unrolled yet, but should this just be `allele`? I don't see an `allele` table that this ID would reference.

Reply:
No, this is just an internally generated but public facing numeric allele identifier from ClinVar. There is no table to be referenced. But I'd prefer to keep this as is.